README.md

```diff
@@ -28,7 +28,7 @@ remotes::install_github("Roche-GSK/admiral")
 
 ## Scope
 
-* Build a toolbox of re-usable functions and utilities to create ADaM datasets using R scripts in a modular manner
+* A toolbox of re-usable functions and utilities to create ADaM datasets using R scripts in a modular manner (an "opinionated" design strategy)
 * Pharmaceutical communities and companies are encouraged to contribute to admiral following the provided programming strategy and modular approach
```

Comment:
Can we link here to the programming strategy?

Reply:
References and Documentation section below this already has that link and gives links in a more appropriate order to read, e.g. the Get Started page first, so i don't think needed to add here as well